### PR TITLE
docs: Provide clearer docs on how to define the template you'd like to use

### PR DESCRIPTION
**Pull request by tomkerkhove**
### Prerequisites

- [x] I have read and understood the `CONTRIBUTING` guide
- [ ] The commit message follows the [conventional commits][cc] guidelines
- [ ] Tests for the changes have been added (for bug fixes/features)
- [x] Docs have been added / updated (for bug fixes/features)

### Description

Provide clearer docs on how to define the template you'd like to use because this is a bit burried in the PowerShell installation docs only.

(just a suggestion)

### CODE REVIEW

**Comment by JanDeDobbeleer** [comment]
@tomkerkhove we're going to have an issue with the instructions depending on how you installed oh-my-posh. As this context is only valid for when you're using the module (and not PowerShell in general) people will still get confused. Can I propose to change to `.mdx` and add a tabbed item that addresses PowerShell, zsh, bash, fish?

**Comment by tomkerkhove** [comment]
Happy to do so but only know for PowerShell - Is that OK as well?

If not, I can see if I can figure out the rest.

**Comment by JanDeDobbeleer** [comment]
@tomkerkhove yes, I'll add the others so then we have a shared commit and you stay contributor ;-)

**Comment by tomkerkhove** [comment]
> you stay contributor ;-)

Meh, don't bother :D

File: docs/docs/themes.mdx [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+If you are using PowerShell, you can display every available theme using the following PowerShell cmdlet.
+
+```powershell
+Get-PoshThemes
+```
+
+Or specify the theme you'd like to use:
+
+```powershell
+Set-PoshPrompt -Theme jandedobbeleer
```
Comment (JanDeDobbeleer) [inline docs/docs/themes.mdx:43]: set-posh prompt is now deprecated, due to that and an aligned logic I'm closing this one.

**Pull request closed without merging**